core/editor/loader/v2/editor-v2-loader.php

```diff
@@ -57,6 +58,17 @@ public function init() {
 			$this->assets_config_provider->load( $package );
 		}
 
+		add_filter( 'elementor/editor/v2/scripts/env', function( $env ) {
+			$env['@elementor/http'] = [
+				'base_url' => rest_url( 'elementor/v1' ),
+				'headers' => [
+					'X-WP-Nonce' => wp_create_nonce( 'wp_rest' ),
+				],
+			];
+
+			return $env;
+		} );
```

Comment:
should we add a `register_hooks` method and call it in the editor itself?

Reply:
Lets talk about it